src/components/ui/Panel.svelte

```diff
@@ -42,10 +42,13 @@
 
 <style>
 .your-report {
-  position: sticky; 
+  position: sticky;
   top: 1em;
   grid-row-start: 1;
 }
+:global(#report-findings .your-report {
+  position: initial;
```

Comment:
maybe use static as it has wider browser support?

Reply:
Agreed and pushed!